doc/api/deprecations.md

```diff
@@ -2652,6 +2652,20 @@ Type: Documentation-only
 The [`crypto.Certificate()` constructor][] is deprecated. Use
 [static methods of `crypto.Certificate()`][] instead.
 
+### DEP0147: `fs.rmdir(path, { recursive: true })`
+<!-- YAML
+changes:
+  - version: REPLACME
+    pr-url: https://github.com/nodejs/node/pull/35562
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+`fs.rmdir(path, { recursive: true })` will throw on nonexistent paths, or when
```

Comment:
This makes it sound like the current version of Node.js will do this. Maybe something more like this to make it clear that's not the case?

```suggestion
In future versions of Node.js, `fs.rmdir(path, { recursive: true })` will throw on nonexistent paths, or when
```

(It will need to be re-wrapped to keep it at 80 characters.)